proposal/calc-functions.md

```diff
@@ -145,6 +188,43 @@ Add the following production:
 
 1: The string `abs(` is matched case-insensitively.
 
+## Operations
+
+### Modulo
+
+Replace [the definition of modulo for numbers] with the following:
+
+[the definition of modulo for numbers]: ../types/number.md#modulo
+
+> Differences are highlighted in bold.
+
+Let `n1` and `n2` be two numbers. To determine `n1 % n2`:
+
+* Let `c1` and `c2` be the result of [matching units] for `n1` and `n2` allowing
+  unitless.
+
+* **If `c2` is infinity and has a different sign than `c2` (including
+  oppositely-signed zero), return NaN with the same units as `c1`.**
+
+  > This matches the behavior of CSS's `mod()` function.
+
+* Let `remainder` be a number whose value is the result of `remainder(c1.value,
```

Comment:
I think this should say, "Let `result` be a number", no? Or where is `result` defined?

(reading the current spec, it looks like that may be an existing bug?)

Reply:
Updated `result` references to be `remainder`.